spec/features/consumer_searches_remote_advice_spec.rb

```diff
@@ -35,6 +35,16 @@
     end
   end
 
+  scenario 'Selecting phone and/or online advice with advice filters' do
+    with_elastic_search! do
+      given_i_am_on_the_rad_landing_page
+      and_firms_providing_various_types_of_remote_services_were_indexed
+      when_i_submit_a_search_selecting_remote_advice_and_types_of_advice
+      then_i_am_shown_filtered_firms_list
+      and_they_are_ordered_alphabetically
```

Comment:
I'm disputing whether this behaviour is correct, regardless of what the story says.